renovate/tests/Main.hs

```diff
@@ -102,7 +102,7 @@ findSpaceForPHDRsTests =
         -- Type           Offset   VirtAddr   PhysAddr   FileSiz MemSiz   Flg Align
         -- LOAD           0x000000 0x00010000 0x00010000 0x0031c 0x0031c  R E 0x10000
         -- LOAD           0x00031c 0x0002031c 0x0002031c 0x00004 0x400008 RW  0x10000
-        Nothing T.@=?
+        ELF.NoAddress T.@=?
```

Comment:
As we discussed on Mattermost, it sounds like this test is returning `FallbackAddress`, which I believe is fine. Here's how I would rewrite the above comment:
```
    -- Values in the following test case are taken from
    -- linked-list.noopt.nostdlib.aarch32.exe. The .bss section has a HUGE buffer in it, which
    -- overlaps with the optimal placement of a new PHDR segment. However, we can still find
    -- a FallbackAddress which is safe assuming the binary has no TLS segment (and it doesn't).
```